consul/resource_consul_keys_test.go

```diff
@@ -31,12 +32,31 @@ func TestAccConsulKeys_basic(t *testing.T) {
 					testAccCheckConsulKeysValue("consul_keys.app", "enabled", "true"),
 					testAccCheckConsulKeysValue("consul_keys.app", "set", "acceptanceUpdated"),
 					testAccCheckConsulKeysRemoved("consul_keys.app", "remove_one"),
+					testAccCheckConsulKeysFlags("test/set", 64),
```

Comment:
Do you know why we can't use the built-in helpers here to check the set? I feel like that could be used here but happy to be shown the other way.

Reply:
Yeah I think I was, but no requirement, I will approve either way (still needs a rebase though)

Reply:
I fixed it, thanks!